xds/src/main/java/io/grpc/xds/ClientXdsClient.java

```diff
@@ -222,79 +234,247 @@ private static LdsUpdate processClientSideListener(Listener listener, boolean pa
       throw new ResourceInvalidException(
           "Could not parse HttpConnectionManager config from ApiListener", e);
     }
+    return LdsUpdate.forApiListener(
+        parseHttpConnectionManager(hcm, parseFilter, true /* forClient */));
+  }
+
+  private LdsUpdate processServerSideListener(Listener proto, boolean parseFilter)
```

Comment:
Same as above. Should the param be called `parseHttpFilters` for accuracy?

Reply:
Right.

Reply:
Updated.